tests/Unit/Email/MailerTest.php

```
@@ -0,0 +1,59 @@
+ * Tests the PSR-4 autoloaded Mailer facade.
+ */
+final class MailerTest extends TestCase {
+
+	/**
+	 * The SWH\Email\Mailer class loads through the PSR-4 autoloader.
+	 *
+	 * @return void
+	 */
+	public function test_mailer_class_is_autoloadable() {
+		$this->assertTrue( class_exists( Mailer::class ), 'SWH\\Email\\Mailer must be autoloadable via composer PSR-4.' );
+	}
```

PHPMD (2.15.0)

[error] The method test_mailer_class_is_autoloadable is not named in camelCase. (undefined)

(CamelCaseMethodName)